src/main/g8/src/test/resources/application-test.conf

```diff
@@ -0,0 +1,3 @@
+include "application"
```

Comment:
Ah, that's because we created our own setting?

Reply:
Yeah, and the typed testkit no longer pulls in `application.conf` by default. So either this or duplicating the config (which may be a better idea in general, so maybe I should change that).